src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -264,16 +271,19 @@ public void iCallTheRedeemVoucherAPIWithExpectedStatusOf(int responseCode) {
 
     @Then("I can assert that redemption was successful by asserting the status in response")
     public void iCanAssertThatRedemptionWasSuccessfulByAssertingTheStatusInResponse() {
-        try {
-            JsonNode rootNode = objectMapper.readTree(redeemVoucherResponseBody);
+        await().atMost(awaitMost, SECONDS).untilAsserted(() -> {
```

Comment:
add assert inside

Reply:
Already assertion is present in try catch block